samples/excel/tables/apply-values-filter.yaml

```diff
@@ -0,0 +1,137 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Apply a values filter
+description: Apply a values filter to a table
+script:
+  content: |
+    $('#create-table').click(createTable);
+    $('#filter-data').click(filterData);
+
+    /** Create a new table with some sample data */
+    function createTable() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                // Queue a command to add a new worksheet
+                sheet = context.workbook.worksheets.add();
```

Comment:
Please follow the setup pattern from https://github.com/WrathOfZombies/samples/blob/master/samples/excel/range/set-get-values.yaml